BukkyOmo/Questioner
# #163659364 implement authorization on endpoints

#### What does this PR do?
This PR:
* Implement Fix of form validation.

#### Description of Task to be completed
* fix validation with form
* In the uservalidation.js, remove lastname, firstname and phonenumber
* Also remove in the database.

#### How should this be manually tested?
* Clone this repository
* Check out to <ch-implement-authorization-on-endpoints-#163659364> branch
* Enter user details on postman and test signup and signin.

#### Any background context you want to provide?
No

#### What are the relevant pivotal tracker stories? (if applicable)
#163659364
